e2e/testcases/status_enablement_test.go

```diff
@@ -63,7 +63,7 @@ func TestStatusEnabledAndDisabled(t *testing.T) {
 			resourceGroupHasNoStatus,
 			testpredicates.HasLabel(common.InventoryLabel, id),
 		},
-		testwatcher.WatchTimeout(120*time.Second))
+		testwatcher.WatchTimeout(180*time.Second))
```

Comment:
Is there any reason to not use `DefaultWaitTimeout`?

Reply:
Switched to `DefaultWaitTimeout`. Thanks for the suggestion.